client/src/pages/index.tsx

```diff
@@ -256,7 +255,13 @@ function IndexPage(props: {
         height: shouldDisplayPortrait() ? "250px" : windowHeight - 100,
       }}
     >
-      <HistoryChat height={windowHeight - 100} />
+      <Chat
+        height={windowHeight - 100}
+        chatProps={{
```

Comment:
these props should not be nested...why do we have a separate `chatProps` inside of the root `chatProps`?

Reply:
One `chatProps` is for chat mentors and the other  `chatProps` for video mentors